diff --git a/ftdipp/CMakeLists.txt b/ftdipp/CMakeLists.txt
index f05bead..57003f2 100644
--- a/ftdipp/CMakeLists.txt
+++ b/ftdipp/CMakeLists.txt
@@ -12,6 +12,7 @@ include_directories(BEFORE ${CMAKE_CURRENT_BINARY_DIR}
 include_directories(${Boost_INCLUDE_DIRS})
 
 # Shared library
+if(BUILD_SHARED_LIBS)
 add_library(ftdipp1 SHARED ${cpp_sources})
 
 math(EXPR VERSION_FIXUP "${MAJOR_VERSION} + 1")    # Compatibility with previous releases
@@ -30,7 +31,7 @@ install ( TARGETS ftdipp1
 				)
 
 # Static library
-if ( STATICLIBS )
+else()
 	add_library(ftdipp1-static STATIC ${cpp_sources})
 	set_target_properties(ftdipp1-static PROPERTIES OUTPUT_NAME "ftdipp1")
 	set_target_properties(ftdipp1-static PROPERTIES CLEAN_DIRECT_OUTPUT 1)
